draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -826,15 +908,17 @@
 
           <t hangText="- Res (4 bits): ">Reserved for future use.</t>
 
+	  <!-- XXX JMH - MT is another case where describing that this is a
+	  4-bit unsigned integer makes it clear what the contents will be -->
           <t hangText="- MT (4 bits): ">Metric Type. This document defines a
           default metric type as value 0, indicating this is the normalized
           metric derived by multiple types of metrics. The rules to derive
           the normalized metric are out of scope of this document and
           defined by per service. Other Metric Types could be defined by other documents in the future.</t>
```

Comment:
is the following wording okay? 

"<t hangText="- MT (4 bits): ">Metric Type _which is a 4-bits unsigned integer._ "

Reply:
See next push for candidate merged text.